src/shared/treeview/webviews/showErrorDetails.ts

```diff
@@ -32,9 +34,13 @@ export async function showErrorDetails(element: ErrorNode) {
         })
     } catch (err) {
         const error = err as Error
+        showResult = 'Failed'
+
         logger.error(error.message)
 
         const baseTemplateFn = _.template(BaseTemplates.SIMPLE_HTML)
         view.webview.html = baseTemplateFn({ content: `Error displaying error details: ${error.message}` })
+    } finally {
+        recordAwsShowExplorerErrorDetails({ result: showResult })
```

Comment:
@awschristou is the goal to make this error webview used for more than just Lambdas? If that's the case, we should also add a metadata field for the type of node that's being inspected.

Reply:
This is supposed to be a 1:1 translation of the existing call. If there is an issue with the data we get here, we can fix it in another PR.

Reply:
I agree with @hunterwerlla on using a separate PR for changing the behavior. I don't recall anything Lambda specific in this webview, so the metric seems reasonable at this time to me -- if we need more granular details about what is surfacing error details this could be revisited.